float64 Fr2joint0
float64 Fr2joint1
float64 Fr2joint2
float64 Fr2joint3
float64 Fr2joint4
float64 Fr2joint5
float64 Fr2joint6
float64 Fr3joint0
float64 Fr3joint1
float64 Fr3joint2
float64 Fr3joint3
float64 Fr3joint4
float64 Fr3joint5
float64 Fr3joint6
bool enable
